extension/js/common/core/mime.ts

```
@@ -177,7 +177,7 @@ export class Mime {
   }
 
   public static decode = (mimeMsg: Uint8Array): Promise<MimeContent> => {
-    return new Promise(async resolve => {
+    return new Promise(resolve => {
```

good catch, that must have been swallowing errors silently!